talend-component-maven-plugin/src/it/web/test/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (C) 2006-2022 Talend Inc. - www.talend.com
+   Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.talend.ci.api-tester</groupId>
+    <artifactId>component_runtime</artifactId>
+    <version>1.0.0</version>
+    <build>
+        <plugins>
+            <plugin>
+
+                <!-- The Talend maven plugin -->
+                <groupId>org.talend.ci</groupId>
+                <artifactId>api-tester-maven-plugin</artifactId>
+                <version>1.10.0</version>
+
+                <executions>
+                    <execution>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <file>component_runtime_prod.json</file>
+
+                            <!-- Enter your account ID here. Consult your account information in Talend Management Console Subscription page. -->
+                            <accountId>${env.API_TESTER_ID}</accountId>
+
+                            <instance>eu</instance>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
```

Comment:
You could l;ink the execution with a profile, this way unless the profile provided it won't be run.
Thinking about next step, having a real it....